docs/topics/proc_creating-a-rhoardoc-jira-release.adoc

```diff
@@ -2,7 +2,7 @@
 [id='creating-a-rhoardoc-jira-release_{context}']
 = Creating a RHOARDOC JIRA release
 
-The JIRA _Release_ mechanism is used to track work between publishings of the documentation.
+The JIRA _Release_ mechanism is used to track work between publishes of the documentation.
```

Comment:
Do we mean to say "between two consecutive versions of the published documentation'?

Reply:
Yes, pretty much. In any case, _publish_ is not a noun.